miio/vacuum_cli.py

```diff
@@ -145,6 +145,29 @@ def consumables(vac: miio.Vacuum):
 
 
 @cli.command()
+@click.argument('name', type=str, required=True)
+@pass_dev
+def reset_consumable(vac: miio.Vacuum, name):
+    """Reset consumable."""
```

Comment:
Could you also add a list information about the available values? Adding an empty line after the "title line" will make it only visible when help on the specific command is requested, IIRC, so I suppose something like this would work:
```
"""Reset consumable state.

Allowed values:
main_brush - explanation
another one - x
```